pkgs/tools/X11/xkb-switch-i3/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv
+, cmake
+, fetchgit
+, i3
+, jsoncpp
+, libsigcxx
+, libX11
+, libxkbfile
+, pkg-config
+}:
+
+stdenv.mkDerivation rec {
+  pname = "xkb-switch-i3";
+  version = "1.8.1";
+
+  src = fetchgit {
+    url = "https://github.com/Zebradil/xkb-switch-i3";
+    rev = version;
+    sha256 = "15c19hp0n1k3w15qn97j6wp5b8hbk0mq6x3xjfn6dkkjfz1fl6cn";
+    fetchSubmodules = true;
```

Comment:
I would prefer if we add a package for https://github.com/drmgc/i3ipcpp instead of using the vendored version to more easily patch it.

Reply:
I am not sure yet how to do that. I am confused by the message from the repo:
> Yet the only way of using is to add this repo as a submodule

Reply:
Then we just keep it this way. I thought this would be easier.